com/copy_ydb_dist_dir.csh

```diff
@@ -0,0 +1,23 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2005-2016 Fidelity National Information		#
+# Services, Inc. and/or its subsidiaries. All rights reserved.	#
+#								#
+# Copyright (c) 2017,2018 YottaDB LLC. and/or its subsidiaries.	#
```

Comment:
The # alignment at the end seems to be off for this line. Tabs vs spaces?

Reply:
I don't seem to have this issue